apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/service/NamespaceBoAsyncService.java

```diff
@@ -0,0 +1,69 @@
+package com.ctrip.framework.apollo.portal.service;
+
+
+import com.ctrip.framework.apollo.common.constants.GsonType;
+import com.ctrip.framework.apollo.common.dto.ItemDTO;
+import com.ctrip.framework.apollo.common.dto.ReleaseDTO;
+import com.ctrip.framework.apollo.portal.environment.Env;
+import com.google.gson.Gson;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.scheduling.annotation.AsyncResult;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Future;
+
+/**
+ * @author renjiahua
+ * @date 2020/12/26
+ */
+
+@Service
+public class NamespaceBoAsyncService {
+    /**
+     * releaseService
+     */
+    @Autowired
+    private ReleaseService releaseService;
```

Comment:
other places has changed to constructor injection, I think we need th follow the same style here